package.json

```diff
@@ -35,13 +35,14 @@
     "eslint-plugin-import": "^2.25.2",
     "eslint-plugin-n": "^15.0.0 || ^16.0.0 ",
     "eslint-plugin-promise": "^6.0.0",
-    "ethers": "^6.4.0",
     "hardhat": "^2.14.0",
     "hardhat-gas-reporter": "^1.0.8",
     "solidity-coverage": "^0.8.0",
     "ts-node": ">=8.0.0",
-    "typechain": "^8.1.0",
     "typescript": ">=4.5.0"
   },
-  "dependencies": {}
+  "dependencies": {
+    "ethers": "5.1.3",
```

Comment:
Did you mean to downgrade from `"ethers": "^6.4.0",` to `"ethers": "5.1.3"`?

I think that may break the existing tests since they're written with ethers v6.

Reply:
It was intentional. I was getting some peer dependency error from yarn. I changed to `ethers v6.8.0` and no longer seeing the error 🤷

Reply:
This package.json still downgrades ethers, do we need to check in a new version?